src/main/java/com/cloudant/client/api/Search.java

```diff
@@ -369,7 +369,12 @@ public Search counts(String[] countsfields) {
     public Search drillDown(String fieldName, String fieldValue) {
         assertNotEmpty(fieldName, "fieldName");
         assertNotEmpty(fieldValue, "fieldValue");
-        uriBuilder.query("drilldown", "[\"" + fieldName + "\",\"" + fieldValue + "\"]");
+        JsonArray drillDownArray = new JsonArray();
+        JsonPrimitive fieldNamePrimitive = new JsonPrimitive(fieldName);
+        drillDownArray.add(fieldNamePrimitive);
+        JsonPrimitive fieldValuePrimitive = new JsonPrimitive(fieldValue);
+        drillDownArray.add(fieldValuePrimitive);
+        uriBuilder.query("drilldown", drillDownArray);

```

Comment:
`drilldown` can be specified multiple times, looks like the previous code was incorrect in this regard.
It should be `uriBuilder.query("drilldown", drillDownArray, false)` to not replace the parameter. I'm guessing this will be impacted by #167 though. Maybe it is worth moving this out into another issue as it is a bit unrelated.